src/sounderpy/calc.py

```diff
@@ -394,6 +395,14 @@ def interpolate(var,hgts,step):
             thermo['dparcel_p'] = ma.masked
             thermo['dparcel_T'] = ma.masked
 
+        #--- PARCEL PROFILES ---#
+        # ---------------------------------------------------------------
+
+        if self.include_all_parcels:
+            parcels = [parcelx(prof, pres=p[i].m, tmpc=T[i].m, dwpc=Td[i].m) for i in range(len(p))]
```

Comment:
Performance overhead is negligible at about 1-2% of the total runtime of `build_sounding()`.